docs/quickstart.md

```diff
@@ -329,17 +323,17 @@ HTTPX will use universal line endings, normalising all cases to `\n`.
 In some cases you might want to access the raw bytes on the response without applying any HTTP content decoding. In this case any content encoding that the web server has applied such as `gzip`, `deflate`, or `brotli` will not be automatically decoded.
 
 ```
->>> async with httpx.stream("GET", "https://www.example.com") as r:
-...     async for chunk in r.aiter_raw():
+>>> with httpx.stream("GET", "https://www.example.com") as r:
+...     for chunk in r.iter_raw():
 ...         print(chunk)
 ```
 
 If you're using streaming responses in any of these ways then the `response.content` and `response.text` attributes will not be available, and will raise errors if accessed. However you can also use the response streaming functionality to conditionally load the response body:
 
 ```
```

Comment:
These need a language for syntax highlighting — we can submit a separate PR though.

```suggestion
```python
```